content/pages/faq.md

```diff
@@ -8,7 +8,8 @@ out]({filename}/pages/contact.md) to us!
 
 ## What does &ldquo;women-only&rdquo; mean? ##
 
-Our women-only events are open to cis and trans women, and non-binary individuals.
+Our women-only events are open to cis and trans women, and non-binary
```

Comment:
it's kinda weird that there were changes about non volunteer stuff in this diff, but I don't think it's a huge deal and it's nice to have the line lengths adjusted better